docs/metrics.md

```diff
@@ -166,39 +166,49 @@ soroban.host-fn-op.read-entry                | meter     | number of entries rea
+soroban.host-fn-op.count                     | meter     | number of total `InvokeHostFunctionOp` operations
 soroban.host-fn-op.exec                      | timer     | time spent in `InvokeHostFunctionOp`
-soroban.config.contract-max-size-bytes           | meter     | soroban config setting `contract_max_size_bytes`
-soroban.config.ledger-max-instructions           | meter     | soroban config setting `ledger_max_instructions`
-soroban.config.tx-max-instructions               | meter     | soroban config setting `tx_max_instructions`
-soroban.config.tx-memory-limit                   | meter     | soroban config setting `tx_memory_limit`
-soroban.config.ledger-max-read-ledger-entries    | meter     | soroban config setting `ledger_max_read_ledger_entries`
-soroban.config.ledger-max-read-bytes             | meter     | soroban config setting `ledger_max_read_bytes`
-soroban.config.ledger-max-write-ledger-entries   | meter     | soroban config setting `ledger_max_write_ledger_entries`
-soroban.config.ledger-max-write-bytes            | meter     | soroban config setting `ledger_max_write_bytes`
-soroban.config.tx-max-read-ledger-entries        | meter     | soroban config setting `tx_max_read_ledger_entries`
-soroban.config.tx-max-read-bytes                 | meter     | soroban config setting `tx_max_read_bytes`
-soroban.config.tx-max-write-ledger-entries       | meter     | soroban config setting `tx_max_write_ledger_entries`
-soroban.config.tx-max-write-bytes                | meter     | soroban config setting `tx_max_write_bytes`
-soroban.config.bucket-list-target-size-bytes     | meter     | soroban config setting `bucket_list_target_size_bytes`
-soroban.config.tx-max-contract-events-size-bytes | meter     | soroban config setting `tx_max_contract_events_size_bytes`
-soroban.config.contract-data-key-size-bytes      | meter     | soroban config setting `contract_data_key_size_bytes`
-soroban.config.contract-data-entry-size-bytes    | meter     | soroban config setting `contract_data_entry_size_bytes`
\ No newline at end of file
+soroban.restore-fprint-op.read-ledger-byte   | meter     | number of entry bytes in read entries
+soroban.restore-fprint-op.write-ledger-byte  | meter     | number of entry bytes in written entries
+soroban.ext-fprint-ttl-op.read-ledger-byte   | meter     | number of entry bytes in read entries
+soroban.ledger.cpu-insn                      | histogram | metered cpu instructions per ledger
+soroban.ledger.read-entry                    | histogram | number of entries read per ledger
+soroban.ledger.read-byte                     | histogram | number of entry bytes in read entries per ledger
+soroban.ledger.write-entry                   | histogram | number of entries written per ledger
+soroban.ledger.write-byte                    | histogram | number of entry bytes in written entries per ledger
+soroban.config.contract-max-rw-key-byte      | counter   | soroban config setting `contract_data_key_size_bytes`
```

Comment:
what is the point of exporting those soroban.config.* as metrics?

Reply:
(I agree they should be `counter` if we keep them)

Reply:
In order to display the limits on top of all of the resource consumption metrics (defined above). Without these we'll have to hardcode them on grafana and display them as variables, which is a static and inconvenient.